pages/common/typeinc.md

```diff
@@ -1,21 +1,21 @@
 # typeinc
 
-> An `ncurses` based terminal typing speed test program, written in python.
+> An `ncurses` based terminal typing speed test program, written in Python.
 > Try out different difficulty levels and improve your typing speed.
 > More information: <https://github.com/AnirudhG07/Typeinc>.
 
 - Enter the typing test:
 
 `typeinc`
 
-- Display the top 10 [r]anklist for input difficulty level:
+- Display the top 10 rank list for input difficulty level:
```

Comment:
@sebastiaanspeck the short/long option placeholder already displays the related meaning, why do we need a mnemonic in this context (which would be ignored in translations)?

Reply:
> I think we are not consistent in other pages, AFAIK I have seen mnemonics and short-long options being used together before

Yes they were (mnemonics were kept for contextual words), but since the introduction of the newer syntax it is much cleaner to use this in shorter examples without extra contextual words than mnemonics.